maintenance/pin.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+set -x
+
+docker run -t -v $PWD:/src -w /src -e SUFFIX=-py2 python:2.7 maintenance/pin-helper.sh "$@"
+docker run -t -v $PWD:/src -w /src python:3.6 maintenance/pin-helper.sh "$@"
```

Comment:
Question: what is the use case for using "$@" here?

Reply:
There probably isn't one. :)